app/views/forum/post.scala

```diff
@@ -27,7 +26,7 @@ object post {
           " ",
           userIdLink(p.userId, withOnline = false),
           " ",
-          span(cls := "extract")(shorten(p.text, 70))
+          span(cls := "extract")(p.text)
```

Comment:
I don't think you want to render the full post text on the homepage.

Reply:
I think this is fixed now?